pkg/controller/datavolume/import-controller_test.go

```diff
@@ -1579,12 +1579,6 @@ var _ = Describe("All DataVolume Tests", func() {
 			Expect(err).ToNot(HaveOccurred())
 			Expect(usePopulator).To(BeFalse())
 		},
-			Entry("Imageio", &cdiv1.DataVolumeSource{
```

Comment:
what about test that make sure you create volumeimportsource for the pvc with the expected checkpoint info?
And also UT for import populator same as done for datavolume that it creates pvc prime with the right annotations (https://github.com/kubevirt/containerized-data-importer/blob/main/pkg/controller/datavolume/import-controller_test.go#L768 for example and there are some more)